bson/raw_bson.py

```diff
@@ -13,6 +13,46 @@
 # limitations under the License.
 
 """Tools for representing raw BSON documents.
+
+Inserting and Retrieving RawBSONDocuments
+=========================================
+
+Example: Moving a document between different databases/collections
+.. code-block:: python
+
+  import bsonjs
+  from pymongo import MongoClient
+  from bson.raw_bson import RawBSONDocument
+
+  client = MongoClient("Localhost", 27017, document_class=RawBSONDocument)
+  db = client.db
+  doc = {"_id": 1, "test": "1"}
+  doc_bson = bsonjs.loads('{"_id": 1, "test": "1"}')
+
+  # add original document to collection
```

Comment:
Suggest changing this to something like:

```
  # add some documents to collection 'test'
  result = db.test.insert_many([
      {'a': 1},
      {'b': 1},
      {'c': 1},
      {'d': 1}])
  assert result.acknowledged

  # duplicate all documents in collection 'test' in collection 'test_copy'
  for doc in db.test.find():
      print(f"raw document: {doc.raw}\ndecoded document: {bson.decode(doc.raw)}")
      db.test_copy.insert_one(doc)
```

Reply:
You should also include the sample output generated from the `print` above in your code sample to illustrate how raw BSON documents work. 
This is just a suggestion - you should write the example the way it makes most sense to you.

Reply:
yep i updated it to use multiple docs